awscli/examples/s3/sync.rst

```diff
@@ -68,7 +68,7 @@ Output::
     upload: test2.txt to s3://mybucket/test2.txt
 
 The following ``sync`` command syncs files under a local directory to objects under a specified prefix and bucket by
```

Comment:
The whole thing could be worded more clearly. Yes, this is downloading from s3 to local, with a sync command. The other sync command examples do say "sync {destination} to {source}" in their descriptions, maybe updating all the examples to say "sync {destination} to match {source}" or something similar would be clearer.

I only updated the description of the mechanism which was obviously wrong, rather than the description of what was trying to be accomplished, since the language used might already describe that (it's hard to be sure).